src/atoms/openFilesState.ts

```diff
@@ -0,0 +1,136 @@
+import { FileEntry } from '@tauri-apps/api/fs';
+import { atom, Getter } from 'jotai';
+
+export type PaneId = 'LEFT' | 'RIGHT';
+export const DEFAULT_PANE: PaneId = 'LEFT';
+type FileId = string;
+
+interface OpenFilesState {
+  files: Record<FileId, FileEntry>;
```

Comment:
This is also my first time with Jotai and I'm exploring the options.
I think it might be a good solution to use atomFamily here (and also maybe for the panels - given that we might have more in the future) but I'm not sure if that should be added right now in this PR as we are also considering Zustand for the state management. Will keep that for an experimental.